tests/test_subfunctions.py

```diff
@@ -31,7 +31,8 @@
 from miscnn.data_loading.interfaces import Dictionary_interface
 from miscnn.data_loading.sample import Sample
 from miscnn.processing.subfunctions import *
-
+from multiprocessing import freeze_support
```

Comment:
why required?

Reply:
Experienced a multiprocessing deadlock when running this test. Uncertain what caused it but this allowed a better shutdown.

Reply:
ok